schema/properties_directory/non-scalar/is_relaxed.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$id": "properties-directory/non-scalar/is-relaxed",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Is relaxed property schema",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "../../material.json"
```

Comment:
Do we need to store material here too?

Reply:
Material exists in all `in_relaxed` properties in the production database